src/ASTBuilder.ts

```diff
@@ -1583,26 +1585,24 @@ export class ASTBuilder
       unitAlias,
       unitAliasIdentifier,
       symbolAliases,
-      symbolAliasesIdentifiers,
+      symbolAliasesIdentifiers
     }
 
     return this._addMeta(node, ctx)
   }
 
   public buildEventParameterList(ctx: SP.EventParameterListContext) {
-    return ctx.eventParameter().map((paramCtx: any) => {
+    return ctx.eventParameter().map((paramCtx: SP.EventParameterContext) => {
       const type = this.visit(paramCtx.typeName())
-      let name = null
-      if (paramCtx.identifier()) {
-        name = this._toText(paramCtx.identifier())
-      }
+      const identifier = paramCtx.identifier()
+      const name = identifier ? this._toText(identifier) : null
```

Comment:
small rebase to avoid calling `paramCtx.identifier()` twice and avoid having to specify the existence of the identifier call the second time **within the if statement**
`this._toText(paramCtx.identifier()!)`